src/main/java/com/SAS/League/League.java

```diff
@@ -150,9 +154,28 @@ public void addPointsPolicy(Season season, PointsPolicy pointsPolicy) {
      * @param rankPolicy the rank policy you want to add
      */
     public void addRankPolicy(Season season, LeagueRankPolicy rankPolicy) {
-        if (this.seasonList.contains(season)&&!this.rankPolicy.containsKey(season)) {
+        if (this.seasonList.contains(season) && !this.rankPolicy.containsKey(season)) {
             this.rankPolicy.put(season, rankPolicy);
             season.addRankPolicy(this, rankPolicy);
         }
     }
+
+    /**
+     *
+     * @return the hashmap of the referees for this league for each season
+     */
+    public HashMap<Season,HashSet<Referee>> getReferees() {
+        return referees;
+    }
+
+    /**
+     * @param season     the season that the  referee is relevant to
+     * @param ref the referee you want to add to the list of referees
+     */
+    public void addReferee(Season season, Referee ref) {
+        if (this.seasonList.contains(season) && !this.referees.containsKey(season)) {
```

Comment:
the condition should be positive for this.referees.containsKey(season) no?
I think you want the referees list to contain the season

Reply:
you're right, must have slipped from a a different function